src/components/UI/card/Card.css

```diff
@@ -3,31 +3,35 @@
 .card {
   composes: base container rounded depth;
   overflow: hidden;
-}
-
-.card > *:not(iframe, video, img, header, footer) {
```

Comment:
At a glance I'm uncertain why these ungainly selectors were added - are you sure we don't need them?